examples/tv-app/android/include/content-launcher/AppContentLauncherManager.cpp

```diff
@@ -53,8 +57,11 @@ void AppContentLauncherManager::HandleLaunchUrl(CommandResponseHelper<LaunchResp
     string displayStringString(displayString.data(), displayString.size());
 
     // TODO: Insert code here
+
+    const char * resStr = mCommandDelegate.sendCommand(mEndpointId, contentUrlString);
```

Comment:
what happens here when the commandDelegate is null (eg. the case with the 4 dummy AppImpls in AppImpl.h)?